src/api/helpers/utils.js

```diff
@@ -15,27 +15,19 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
 import {get as _get} from 'lodash';
 import {loadEntity} from './middleware';
-
-
```

Comment:
These two lines shouldn't be removed, it created an ESLint warning
"Expected 2 empty lines after import statement not followed by another import (import/newline-after-import)"